pkg/comp-functions/functions/common/release.go

```diff
@@ -108,5 +109,9 @@ func NewRelease(ctx context.Context, svc *runtime.ServiceRuntime, comp InfoGette
 		},
 	}
 
+	if strings.HasPrefix(svc.Config.Data["chartRepository"], "oci://") {
+		release.Spec.ForProvider.Chart.URL = fmt.Sprintf("%s:%s", svc.Config.Data["chartRepository"], svc.Config.Data["chartVersion"])
+	}
+
```

Comment:
small workaround to make Forgejo chart pulling working

Reply:
Do OCI images require the version to be part of the url?